### PR TITLE
Develop->Master Merge

### PR DESCRIPTION
**Pull request by liushuyu**
This change is [Reviewable](https://reviewable.io/reviews/aosc-dev/website-site-ng/81)

### CODE REVIEW

**Comment by liushuyu** [comment]
Note: BitHound code style check relaxed, trivial errors are ignored now.

**Comment by liushuyu** [comment]
All checks are passed, code coverage increased to 87%. Awaiting @LionNatsu to approve the changes to apply to  `master` branch

File: views/mirror-status.pug [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        td #{moment(params.repo_info.lack).format('YYYY-MM-DD HH:mm:ss')} (#{moment(params.repo_info.lack).twix(Date.now()).humanizeLength()} ago)
+        td #{params.repo_info.status} (#{params.repo_info.status_desc})
+      for mirror in params.mirrors
+        +mirror-entry(mirror, params.repo_info)
+  p
+    a.btn.btn-primary#gen-btn(href="javascript:void(0);", onclick="startMeasurement();") Generate APT Configuration
+    span(style="display: none;")#pdesc Crunching mirrors information for you, just a moment...!
+    div.progress(style="display: none;")#pbar
+      div.progress-bar.progress-bar-striped.active#pgs(role="progressbar", aria-valuenow="70", aria-valuemin="0", aria-valuemax="100", style="width:0%") 0%
+  br
```
Comment (LionNatsu) [inline views/mirror-status.pug]: Why did you break a line at the end of ...page?

File: test.js [hunk 2]
```diff
@@ -28,22 +40,22 @@ function getSlug(url, ext, callback) {
 
 request(url).get('/').end(function(err, res) {
     if (err) {
-        require('./app.js').app;
+        const app = require('./app.js').app;
```
Comment (liushuyu) [inline test.js]: This is just to suppress linter warnings.

**Comment by liushuyu** [comment]
I don't know if it's safe to assume the user who uses this page is using a(n) ES 2016 compatible browser, since `let`, arrow functions, and template literals are not supported in old browsers.

**Comment by LionNatsu** [comment]
> I don't know if it's safe to assume the user who uses this page is using a(n) ES 2016 compatible browser, since let, arrow functions, and template literals are not supported in old browsers.

Sorry, I didn't notice the path of script... Can you revert this...?

**Comment by liushuyu** [comment]
>Sorry, I didn't notice the path of script... Can you revert this...?

Working on this, will rebase the repository to revert this commit partially.

**Comment by liushuyu** [comment]
Merged #81

**Pull request merged**